packages/app-project/src/components/AuthModal/components/LoginForm/components/Form/Form.spec.js

```diff
@@ -41,7 +47,10 @@ describe('Component > Form', function () {
   describe('when being edited', function () {
     it('should call handleChange on the inputs', function () {
       const handleChangeSpy = sinon.spy()
-      const wrapper = shallow(<Form handleChange={handleChangeSpy} />)
+      const wrapper = shallow(
+        <Form handleChange={handleChangeSpy} />,
+        { wrappingComponent: <Grommet />, wrappingComponentProps: { theme: zooTheme } }
```

Comment:
Maybe declare this as a constant too, since you seem to be re-using it in a bunch of tests.